### PR TITLE
MDS-4217 - remove frontend references from build.yaml, add separate build file for frontend

### PR DESCRIPTION
**Pull request by wilson496**
# Main

- What did you focus on in this PR?
- Keep all headers and just leave a single list item 'N/A' if there's no content for that header, otherwise remove N/A and add your details!

# Other

- What other side fixes, tweaks did you add on the side?
- N/A

# How to test

- N/A

# Notes

- Any notes/screenshots for this PR?
- N/A

### CODE REVIEW

File: .github/workflows/build-frontend.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: CORE API - Build & Deploy To DEV
```
Comment (vyasworks) [inline .github/workflows/build-frontend.yaml]: We should rename this to core-webapp

File: .github/workflows/build-frontend.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+    paths:
+      - services/core-web/**
+      - .github/workflows/build-frontend.yaml
```
Comment (vyasworks) [inline .github/workflows/build-frontend.yaml]: file rename here for semantic accuracy

File: .github/workflows/build-frontend.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+env:
+  INITIAL_TAG: latest
+  TAG: dev
+  NAME: frontend
+  CONTEXT: services/core-web/
+
+jobs:
+  build-backend:
```
Comment (vyasworks) [inline .github/workflows/build-frontend.yaml]: rename here for semantic accuracy

File: .github/workflows/build-frontend.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    branches:
+      - develop
+    paths:
+      - services/core-web/**
+      - .github/workflows/build-frontend.yaml
+
+env:
+  INITIAL_TAG: latest
+  TAG: dev
+  NAME: frontend
```
Comment (vyasworks) [inline .github/workflows/build-frontend.yaml]: rename here for semantic accuracy

File: .github/workflows/build-frontend.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+          docker build -t ${{ env.NAME }}:${{ env.INITIAL_TAG }} ${{ env.CONTEXT }} -f ${{ env.CONTEXT }}Dockerfile.ci
+          docker tag ${{ env.NAME }}:${{ env.INITIAL_TAG }} ${{ secrets.CLUSTER_REGISTRY }}/${{ secrets.NS_TOOLS }}/${{ env.NAME }}:${{ env.INITIAL_TAG }}
+          docker tag ${{ env.NAME }}:${{ env.INITIAL_TAG }} ${{ secrets.CLUSTER_REGISTRY }}/${{ secrets.NS_TOOLS }}/${{ env.NAME }}:${{ env.TAG }}
+      - name: Push
+        run: |
+          docker push --all-tags ${{ secrets.CLUSTER_REGISTRY }}/${{ secrets.NS_TOOLS }}/${{ env.NAME }}
+
+  # trigger-gitops:
+  #   runs-on: ubuntu-20.04
+  #   needs: build-frontend
```
Comment (vyasworks) [inline .github/workflows/build-frontend.yaml]: previos job name is build-backend. This should be renamed too

**Comment by vyasworks** [comment]
I would like to put a pause on this until we have some stability on overall pipeline and features are being delivered. 

Another approach I want to implement is to have parallel implementation

where both the argo build + deploy and this work and once we are happy with argo deploys, we can deprecate this.

File: .github/workflows/build-frontend.yaml (hunk 1 above)
Comment (JustinMacaulay) [reply .github/workflows/build-frontend.yaml]: shouldn't this just be core-web?

**Review by vyasworks — DISMISSED** [review]

**Review by vyasworks — DISMISSED** [review]

**Review by vyasworks — APPROVED** [review]

**Pull request merged**